pkg/products/solutionexplorer/reconciler.go

```diff
@@ -217,6 +238,31 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+func (r *Reconciler) reconcileConsoleLink(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	cl := &consolev1.ConsoleLink{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "rhmi-solution-explorer",
+		},
+		Spec: consolev1.ConsoleLinkSpec{
+			ApplicationMenu: &consolev1.ApplicationMenuSpec{},
+		},
+	}
+
+	_, err := controllerutil.CreateOrUpdate(ctx, serverClient, cl, func() error {
+		cl.Spec.ApplicationMenu.ImageURL = "https://avatars1.githubusercontent.com/u/41683098"
+		cl.Spec.ApplicationMenu.Section = "Red Hat Applications"
+		cl.Spec.Href = r.Config.GetHost()
+		cl.Spec.Location = consolev1.ApplicationMenu
+		cl.Spec.Text = "Solution Explorer"
+		return nil
+	})
+	if err != nil {
+		return integreatlyv1alpha1.PhaseFailed, errors.Wrap(err, "error creating or updating solution explorer console link")
```

Comment:
Same, just would be nice to use `fmt.Errorf` instead of the errors package

Reply:
thanks, removed